gpt_handler.py

```diff
@@ -1,28 +1,31 @@
 from os import environ
 import openai
 from dotenv import load_dotenv
-from telegram.ext import CallbackContext
-
 from consts import SUMMARIZER_ROLE
+from typing import List, Dict
 
 
 def configure_gpt_api_key():
     load_dotenv()
     openai.api_key = environ.get("OPEN_AI")
 
 
-async def get_reply_from_chatgpt(context: CallbackContext) -> str:
+async def get_reply_from_chatgpt(messages_history: List[Dict[str, str]]) -> str:
     response = openai.ChatCompletion.create(
```

Comment:
this is an async function, for making this function to be async you must use acreate, not create!